umap/static/umap/js/umap.core.js

```diff
@@ -296,6 +296,18 @@ L.Util.normalize = function (s) {
     .replace(/[\u0300-\u036f]/g, '')
 }
 
+L.Util.parseDateField = function (value) {
```

Comment:
`parseDate` ? Not sure what "field" is in this context :)

Reply:
There is no particular reason for the Field, it just sounded right. But I really don't have any strong feeling on the naming here. ;)

Reply:
Function became obsolete. Decision was made to only support data types that are covered by the Date() constructor itself.